build-tools/packages/version-tools/src/schemes.ts

```diff
@@ -131,12 +135,12 @@ export function bumpVersionScheme(
 		case "virtualPatch": {
 			if (isVersionBumpType(bumpType)) {
 				const translatedVersion = bumpVirtualPatchVersion(bumpType, sv);
-				if (!isVersionBumpType(translatedVersion)) {
-					return translatedVersion;
-				} else {
+				if (isVersionBumpType(translatedVersion)) {
 					throw new Error(
 						`Applying virtual patch failed. The version returned was: ${translatedVersion}`,
 					);
+				} else {
```

Comment:
I thought this was also a flagged issue since the "true" case never executes past throw. So `else` isn't needed.

Reply:
That may be the case for the strict config and not recommended. I'm not sure. But can definitely clean up.